src/deriver/consumer.py

```
@@ -45,36 +44,41 @@
-                    payload,
-                )
-                raise ValueError(f"Invalid payload structure: {str(e)}") from e
-            await deriver.process_representation_task(db, validated)
+        try:
+            validated = RepresentationPayload(**payload)
+        except ValidationError as e:
+            logger.error(
+                "Invalid representation payload received: %s. Payload: %s",
+                str(e),
+                payload,
+            )
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)